app/services/db/user.ts

```diff
@@ -305,3 +317,45 @@ export async function enrichMembersWithEmail(data: PublicCloudRequestData | Priv
 
   return data;
 }
+
+function extractUserIdsFromData(data) {
+  if (!data) return [];
+  return [
+    data.projectOwnerId,
+    data.primaryTechnicalLeadId,
+    data.secondaryTechnicalLeadId,
+    data.expenseAuthorityId,
+    ...(data.members || []).map((member: PublicCloudProductMember) => member.userId),
+  ].filter(Boolean);
+}
+
+export async function enrichSingleUserFields(data) {
```

Comment:
not sure we will need this if the API endpoint does not need the original and decision data